tests/test_expt_file_counts_handler.py

```diff
@@ -60,6 +60,7 @@ def test_get_expt_file_counts_dict():
                         'exact': 's3_example_bucket',
                     },
                 },
+                'forecast_hour': 120
```

Comment:
just curious, why this add?

Reply:
So, filters are based on the table which the information is coming from, the other filters in here are from different tables not the file counts table itself. So I added the forecast_hour as a filter that has to do with file counts table directly so that we are exercising all the code paths, including and especially the file counts table directly.